.github/workflows/karate.yml

```diff
@@ -0,0 +1,114 @@
+name: Integration Tests
+
+on:
+  schedule:
```

Comment:
also, is this running on the main branch by default ?

Reply:
I have no idea

Reply:
It seems to be the default behavior, yes :

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule